doc.md

```diff
@@ -755,6 +756,10 @@ When this option is enabled, search command patterns are considered as globs, ot
 With globbing, `*` matches any sequence, `?` matches any character, and `[...]` or `[^...]` matches character sets or ranges.
 Otherwise, these characters are interpreted as they are.
 
+## globfilter (bool) (default false)
```

Comment:
I think these are supposed to be in alphabetical order, so this should be placed above `globsearch`. The ordering is a bit unfortunate though, maybe you can change the description to something along the lines of "Patterns are treated as globs for the filter command, see globsearch for more details".